src/client/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ set(libclient_srcs
   MetaSession.cc
   Trace.cc
   posix_acl.cc
-  Delegation.cc)
+  Delegation.cc
+  ${CMAKE_SOURCE_DIR}/src/mds/MDSAuthCaps.cc)
```

Comment:
Could this source be in a common directory?

Reply:
Yeah, sounds good to me. the `mon/`, `mds/` and `client/` will use it. I will fix this.